tests/provision_test.go

```diff
@@ -178,6 +193,7 @@ func volumeCreationTest() {
 	By("Running block volume creation test", blockVolCreationTest)
 	By("Running thin volume creation test", thinVolCreationTest)
 	By("Running leak protection test", leakProtectionTest)
+	By("Running shared LV creation test", sharedVolumeTest)
```

Comment:
It is not very clear what this is trying to test. Maybe we could be a bit more verbose on the test title here?
```suggestion
	By("Running LV shared by two pods on the same node test", sharedVolumeTest)
```

Reply:
Updated